pkg/kapp/clusterapply/waiting_changes.go

```diff
@@ -119,7 +120,11 @@ func (c *WaitingChanges) WaitForAny() ([]WaitingChange, error) {
 		}
 
 		if time.Now().Sub(startTime) > c.opts.Timeout {
-			return nil, fmt.Errorf("Timed out waiting after %s", c.opts.Timeout)
+			var timedOutResources []string
+			for _, change := range c.trackedChanges {
+				timedOutResources = append(timedOutResources, change.Graph.Change.Resource().Description())
+			}
+			return nil, fmt.Errorf("Timed out waiting after %s for resources:\n %s", c.opts.Timeout, strings.Join(timedOutResources, "\n "))
```

Comment:
instead of using a new line here, may be we should wrap in `[%s]` so that final error formatting does something like this: https://github.com/cppforlife/go-cli-ui/blob/master/errors/semi_structured_error_test.go

Reply:
I feel listing all resources using new line will be more user readable instead of printing in  an array with comma.

Reply:
updated